lib/dash.coffee

```diff
@@ -55,7 +57,17 @@ plugin = module.exports =
       path = activeEditor.getPath()
       language = activeEditor.getGrammar().name
 
-    spawn('open', ['-g', @createLink(string, path, language)])
+    if platform == "win32"
+      # Exec is used because spawn escapes arguments that contain double-quotes
+      # and replaces them with backslashes. This interferes with the ability to
+      # properly create the child process in windows, since windows will barf
+      # on an ampersand that is not contained in double-quotes.
+      quotedLink = '"' + @createLink(string, path, language) + '"'
+      exec('cmd.exe /c start "" ' + quotedLink, null)
```

Comment:
I lost 2 hours trying to figure out why ```spawn``` wasn't working and troubleshooting a workaround to the issue.  At that point, I tried StackOverflow and found other people complaining about the same issue with double-quotes and spawn, but all the solutions advised to avoid the use of double-quotes or to omit them. 

Unfortunately, that's not really possible here because the double-quotes are necessary so that the command interpreter behaves itself.   That made me crack open the node.js source to figure out what was going wrong<sup>[2]</sup>.  Basically, node tries to UTF8 encode the arguments which causes issues with windows, since the command interpreter uses the local ASCII code page.  The result is double-quote becoming a backslash.

I'm not a fan of having to use different functionality here, but I don't think there is a work around other than to avoid ```spawn``` and use ```exec``` instead. 

[2]: https://github.com/joyent/node/blob/d13d7f74d794340ac5e126cfb4ce507fe0f803d5/src/process_wrap.cc#L182-L194